addon/components/frost-select.js

```diff
@@ -368,7 +368,9 @@ export default Component.extend({
       const onChange = this.get('onChange')
 
       if (typeOf(onChange) === 'function') {
-        this.onChange(selectedValue)
+        run.next(() => {
```

Comment:
This has been causing deprecation warnings for a while in Ember 2.8, Ember 2.10 actually throws a hard error so I was finally forced to address this.